### PR TITLE
feat: update claude-review workflow to use slash command

### PR DESCRIPTION
**Pull request by ashwin-ant**
- Update trigger types to include synchronize, ready_for_review, reopened
- Add pull-requests: write permission for tracking comments
- Replace direct_prompt with /review-pr slash command using custom command file
- Update to use claude-code-action@v1
- Switch to inline comment tool for more precise PR feedback

🤖 Generated with [Claude Code](https://claude.ai/code)

### CODE REVIEW

File: .claude/commands/review-pr.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+   - Verify adequate test coverage
+   - Review test quality and edge cases
+   - Check for missing test scenarios
+
+5. **Documentation**
+   - Ensure code is properly documented
+   - Verify README updates for new features
+   - Check API documentation accuracy
+
+Use a subagent for each area. Instruct each to only provide noteworthy feedback. Once they finish, review the feedback and post only the feedback that you also deem noteworthy.
```
Comment (km-anthropic) [inline .claude/commands/review-pr.md]: Maybe worth to actually create different sub agents to drill the point home

**Review by km-anthropic — CHANGES_REQUESTED** [review]

**Review by km-anthropic — APPROVED** [review]
tysm for making these changes
